frontend/Taskfile.yml

```diff
@@ -71,7 +71,7 @@ tasks:
     cmd: pnpm {{.CLI_ARGS}}
 
   build-viewer-app:
-    deps: [viewer:build-app]
+    deps: [viewer:build]
     label: vite
   watch-viewer-app:
     deps: [viewer:watch-app]
```

Comment:
Probably want to also change

- `build-viewer-app:` to `build-viewer:`
- `watch-viewer-app:` to `watch-viewer:`
- `deps: [viewer:watch-app]` to `deps: [viewer:watch]`